.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint
+pnpm run lint
```

Comment:
For some reason, I encountered some package manager conflict during the pre-commit hook and couldn't create a commit. It was invoking yarn, but then it'd fail with an error that the package manager used in the project is pnpm. I couldn't find yarn used anywhere in the project, so I guess pnpm is the one to use here